Atomic.md

```diff
@@ -0,0 +1,618 @@
+
+While this can sort of be done by submitting multiple transactions with the same sequence number, there is no guarantee that the transactions are processed in the same order they are sent.
+
+#### 2.2.3. `BATCH`
+All transactions will be applied until the first failure, and all transactions after the first failure will not be applied.
+
+#### 2.2.4. `INDEPENDENT`
+All transactions will be applied, regardless of failure.
+
+While this can be achieved with tickets right now, adding it as an atomicity type allows a user to have this functionality without needing the extra reserve.
+
+### 2.3. `RawTransactions`
+
+`RawTransactions` contains the list of transactions that will be applied. There can be up to 8 transactions included. These transactions can come from one account or multiple accounts.
+
+Each inner transaction:
+* **Must** contain an `AtomicTxn` field (see section 3 for details).
+* **Must not** have a sequence number. It must use a sequence number value of `0`.
+* **Must not** have a fee. It must use a fee value of `"0"`.
+* **Must not** be signed (the global transaction is already signed by all relevant parties). They must instead have an empty string (`""`) in the `SigningPubKey` and `TxnSignature` fields.
+* **Must** include an `AtomicTxn` section (described in section 3)
+
+A transaction will be considered a failure if it receives any result that is not `tesSUCCESS`.
+
+**This field is not included in the validated transaction**, since all transactions are included separately as a part of the ledger.
+
+### 2.4. `TxnIDs`
+
+`TxnIDs` contains a list of the transaction hashes/IDs for all the transactions contained in `RawTransactions`. This is the only part of the inner transactions that is saved as a part of the ledger within the `Atomic` transaction, since the inner transactions themselves will be their own transactions on-ledger. The hashes in `TxnIDs` **must** be in the same order as the raw transactions in `RawTransactions`.
+
+While this field seems complicated/confusing to work with, it can easily be abstracted away (e.g. as a part of autofilling) in tooling, and it's easy for `rippled` to check a hash doesn't match its corresponding transaction in `RawTransaction`.
+
+### 2.5. `AtomicSigners`
+
+This field operates similarly to [multisign](https://xrpl.org/docs/concepts/accounts/multi-signing/) on the XRPL. It is only needed if multiple accounts' transactions are included in the `Atomic` transaction; otherwise, the normal transaction signature provides the same security guarantees.
+
+|FieldName | Required? | JSON Type | Internal Type |
+|:---------|:-----------|:---------------|:------------|
+|`Account`|✔️|`string`|`STAccount`|
+|`SigningPubKey`| |`string`|`STBlob`|
+|`Signature`| |`string`|`STBlob`|
+|`Signers`| |`array`|`STArray`|
+
+#### 2.5.1. `Account`
+
+This is an account that has at least one inner transaction.
```

Comment:
I'm thinking about this as having two types of signatures, the _transaction_ signature and the _atomic_ signature. Both are `SigningPubKey`/`TxnSignature` in single account atomics. In multi-account atomics, the transaction signature is the standard `SigningPubKey`/`TxnSignature`, and `AtomicSigners` is the atomic signature. So if the outer `Account` is one of the inner accounts, then it would need two signatures.

Reply:
That seems like a pretty big waste of space, and compute time. I think that the outer account signs the whole transaction, and isn't included on the `AtomicSigners` list.

It looks like you added something similar.